config/controller.yaml

```diff
@@ -77,6 +77,9 @@ spec:
           value: tekton.dev/triggers
         - name: METRICS_PROMETHEUS_PORT
           value: "9000"
```

Comment:
@dibyom we use this env while creating Knative service and Knative itself have metrics exposed on `9090` in order to avoid conflict i have changed to `9000` as part of this PR https://github.com/tektoncd/triggers/pull/1086 (given details over the PR description)

But i think forgot to change for  webhook